### ISSUE TITLE
Toolbar is incorrectly marked up

### ISSUE THREAD

**Issue by Comandeer** [ckeditor/ckeditor5#1404]
## Is this a bug report or feature request? (choose one)

🐞 Bug report

## 💻 Version of CKEditor

https://ckeditor.com/docs/ckeditor5/latest/examples/builds/classic-editor.html

## 📋 Steps to reproduce

1. Inspect editor's toolbar.

## ✅ Expected result

Toolbar has proper label and `[role=toolbar]`.

## ❎ Actual result

Toolbar does not have label and `[role]`.

## 📃 Other details that might be useful

https://w3c.github.io/aria-practices/examples/toolbar/toolbar.html

**Comment by msamsel** [comment]
@Comandeer maybe you could help with creating names for toolbar's label.
There are 3 types of toolbars available:
1. Classic: https://ckeditor.com/docs/ckeditor5/latest/examples/builds/classic-editor.html
2. Balloon Toolbar: https://ckeditor.com/docs/ckeditor5/latest/examples/builds/balloon-editor.html
3. Balloon-Block Toolbar: https://ckeditor.com/docs/ckeditor5/latest/examples/builds/balloon-block-editor.html

Should those toolbars have different names? Maybe with your experience in accessibility, you have some idea how those Toolbars could be named? I appreciate any help with that matter.

**Comment by Comandeer** [comment]
I do not see a need to create different labels for each toolbar types – it would be needed only in a case of having more than one toolbar inside  one editor.

Toolbar's label can be really generic, just to communicate that it's an editor's toolbar. ARIA Practices propose "Text formatting".
